cf-deployment.yml

```diff
@@ -1568,8 +1562,6 @@ instance_groups:
   instances: 2
   vm_type: minimal
   stemcell: default
-  update:
-    serial: true
```

Comment:
My best guess about the `serial: true` here is that it ensures that the `log-api` instance group happens after the `doppler` instance group.

I performed a deploy from scratch where I move the position of `log-api` instance group before the `doppler` instance group, but left `serial: true` enabled. The deploy was successful, showing that `log-api` instance group has no dependency on the `doppler` instance groups and it should be safe to deploy them in parallel.